assets/src/edit-story/components/checklist/priorityChecks.js

```diff
@@ -36,6 +37,7 @@ export function PriorityChecks() {
       <StoryPosterPortraitSize />
       <StoryPosterAspectRatio />
       <PublisherLogoSize />
+      <VideoOptimization />
```

Comment:
Do you think this needs a new flag cuz of the bulk functionality that is new?

Reply:
I don't think so since it's technically the same feature. We might want to include a warning due the resource usage? Maybe a tooltip around the button when multiple videos are going to be optimized?

Reply:
sounds good.
let's not add an additional tooltip outside of the design specs.

Reply:
I think Max is right, it's good separation of concerns - can you give this a new experiment? something about `checklist video optimization`